gtk4/Gir.toml

```diff
@@ -2350,16 +2350,6 @@ manual_traits = ["WidgetExtManual"]
     doc_trait_name = "WidgetExtManual"
     # remove after https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/2887
     [[object.function]]
-    name = "get_css_name"
-        [object.function.return]
-        nullable = true
-    # remove after https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/2887
-    [[object.function]]
-    name = "get_name"
```

Comment:
Can't this be renamed? I thought we added a gir configuration for that in the meantime @GuillaumeGomez ?

Reply:
Euh... Maybe? Sorry, I don't remember at all...

Reply:
`rename =` works